fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -120,6 +176,13 @@ export const LocationSuggest = forwardRef<HTMLInputElement, Props>(
             tone="critical"
           />
         )}
+        {nearestLocationsError && (
```

Comment:
In my opinion, I think we could refactor this to just use the existing `FieldMessage` for `suggestError`. Could combine them and have a general error: "Error fetching location, please try again". Both of these errors would be SEEK API Graph errors and fetching the location (either for lat/long or for autosuggest).

Reply:
Yeah sure, would you prefer doing something like 

`       {(suggestError|| nearestLocationsError) && (
          <FieldMessage
            id="suggestError"
            message="Error fetching location, please try again"
            tone="critical"
          />
        )}`

Reply:
☝️ That merges both errors into a general error message thrown back